### ISSUE TITLE
Wrong URL jump

### ISSUE THREAD

**Issue by tao12345666333** [vmware/octant#238]
![2019-09-03-134337_2338x872_scrot](https://user-images.githubusercontent.com/3264292/64146820-ea974b00-ce50-11e9-83ab-4ffd6e512cd0.png)

My deploy has 3 replicas. Like this:

```
(MoeLove) ➜  ~ kubectl -n work get all -l app=frontend
NAME                                  READY   STATUS    RESTARTS   AGE
pod/saythx-frontend-754f6bc45-5bj58   1/1     Running   0          29m
pod/saythx-frontend-754f6bc45-6b8ww   1/1     Running   0          41m
pod/saythx-frontend-754f6bc45-6jdkv   1/1     Running   0          29m

NAME                      TYPE       CLUSTER-IP      EXTERNAL-IP   PORT(S)        AGE
service/saythx-frontend   NodePort   10.97.225.228   <none>        80:31533/TCP   41m

NAME                              READY   UP-TO-DATE   AVAILABLE   AGE
deployment.apps/saythx-frontend   3/3     3            3           41m

NAME                                        DESIRED   CURRENT   READY   AGE
replicaset.apps/saythx-frontend-754f6bc45   3         3         3       41m

```

When I click the pod on the right, the URL will change to a wrong address.(from `work` namespace to `default` namespace)

![2019-09-03-134351_2386x747_scrot](https://user-images.githubusercontent.com/3264292/64146826-eff49580-ce50-11e9-9019-22af07deea37.png)

octant server will get some error logs.

```
2019-09-03T13:43:46.708+0800    ERROR   event/generator.go:103  event generator error   {"err": "loading object with CacheKey[APIVersion='v1', Kind='Pod']: pods \"saythx-frontend-754f6bc45-5bj58\" not found", "err
Verbose": "pods \"saythx-frontend-754f6bc45-5bj58\" not found\nloading object with CacheKey[APIVersion='v1', Kind='Pod']\ngithub.com/vmware/octant/internal/describer.(*Object).Describe\n\t/go/src/github.com/vmware
/octant/internal/describer/object.go:84\ngithub.com/vmware/octant/internal/modules/overview.(*realGenerator).Generate\n\t/go/src/github.com/vmware/octant/internal/modules/overview/generator.go:93\ngithub.com/vmwar
e/octant/internal/modules/overview.(*Overview).Content\n\t/go/src/github.com/vmware/octant/internal/modules/overview/overview.go:229\ngithub.com/vmware/octant/internal/event.(*ContentGenerator).generateContent\n\t
/go/src/github.com/vmware/octant/internal/event/content_generator.go:75\ngithub.com/vmware/octant/internal/event.(*ContentGenerator).Event\n\t/go/src/github.com/vmware/octant/internal/event/content_generator.go:71
\ngithub.com/vmware/octant/internal/event.runGenerator\n\t/go/src/github.com/vmware/octant/internal/event/generator.go:89\ngithub.com/vmware/octant/internal/event.Stream.func1\n\t/go/src/github.com/vmware/octant/i
nternal/event/generator.go:54\nruntime.goexit\n\t/usr/local/go/src/runtime/asm_amd64.s:1337"}
github.com/vmware/octant/internal/event.runGenerator
        /go/src/github.com/vmware/octant/internal/event/generator.go:103
github.com/vmware/octant/internal/event.Stream.func1
        /go/src/github.com/vmware/octant/internal/event/generator.go:54
2019-09-03T13:43:55.746+0800    ERROR   event/generator.go:103  event generator error   {"err": "loading object with CacheKey[APIVersion='v1', Kind='Pod']: pods \"saythx-frontend-754f6bc45-5bj58\" not found", "err
Verbose": "pods \"saythx-frontend-754f6bc45-5bj58\" not found\nloading object with CacheKey[APIVersion='v1', Kind='Pod']\ngithub.com/vmware/octant/internal/describer.(*Object).Describe\n\t/go/src/github.com/vmware
/octant/internal/describer/object.go:84\ngithub.com/vmware/octant/internal/modules/overview.(*realGenerator).Generate\n\t/go/src/github.com/vmware/octant/internal/modules/overview/generator.go:93\ngithub.com/vmwar
e/octant/internal/modules/overview.(*Overview).Content\n\t/go/src/github.com/vmware/octant/internal/modules/overview/overview.go:229\ngithub.com/vmware/octant/internal/event.(*ContentGenerator).generateContent\n\t
/go/src/github.com/vmware/octant/internal/event/content_generator.go:75\ngithub.com/vmware/octant/internal/event.(*ContentGenerator).Event\n\t/go/src/github.com/vmware/octant/internal/event/content_generator.go:71
\ngithub.com/vmware/octant/internal/event.runGenerator\n\t/go/src/github.com/vmware/octant/internal/event/generator.go:89\ngithub.com/vmware/octant/internal/event.Stream.func1\n\t/go/src/github.com/vmware/octant/i
nternal/event/generator.go:54\nruntime.goexit\n\t/usr/local/go/src/runtime/asm_amd64.s:1337"}
github.com/vmware/octant/internal/event.runGenerator
        /go/src/github.com/vmware/octant/internal/event/generator.go:103
github.com/vmware/octant/internal/event.Stream.func1
        /go/src/github.com/vmware/octant/internal/event/generator.go:54
```

**Comment by wwitzel3** [comment]
@bryanl do you think this might be fixed by the namespace / URL changes you've been working on?

**Comment by bryanl** [comment]
Yes. Octant is changing to have a definitive source for the current namespace in any given context. This will be fixed in 0.7.

**Comment by wwitzel3** [comment]
Fixed in master with #239. Will be in next release.

**Comment by tao12345666333** [comment]
Thanks all.
